app/helpers/middleware_summary_helper.rb

```diff
@@ -9,6 +9,10 @@ def textual_nativeid
     @record.nativeid
   end
 
+  def textual_status
+    @record.status
```

Comment:
I think translation is needed, similar to the below - how tags are handled.
@mith @karelhala is that indeed so?